src/autocomplete/Autocompleter.ts

```diff
@@ -82,15 +82,22 @@ export default class Autocompleter {
         });
     }
 
-    async getCompletions(query: string, selection: ISelectionRange, force = false): Promise<IProviderCompletions[]> {
+    // eslint-disable-next-line
```

Comment:
is this to skirt the line length limit? can we not just break it up? we tend to move each argument onto its own line in cases like this